app/src/main/java/com/github/se/wanderpals/model/repository/TripsRepository.kt

```diff
@@ -1353,8 +1355,35 @@ open class TripsRepository(
       withContext(dispatcher) {
         try {
           Log.d("TripsRepository", "deleteTrip: Deleting trip")
-          removeTripId(tripId) // remove the trip from the user
-          tripsCollection.document(tripId).delete().await() // delete a given trip by its tripId
+
```

Comment:
Great use of multiple launch calls, didn't know it was possible. Nice that it's concurrent.